From 8113d9275f87a5b962cc6e83060577ed36e4c7ca Mon Sep 17 00:00:00 2001
From: Alex Williamson <alex.williamson@redhat.com>
Date: Tue, 4 Mar 2014 15:42:18 +0100
Subject: [PATCH 5/7] Revert: Remove 2 second startup wait.

RH-Author: Alex Williamson <alex.williamson@redhat.com>
Message-id: <20140304154218.6292.69336.stgit@bling.home>
Patchwork-id: 58009
O-Subject: [RHEL7 ipxe PATCH v2 1/3] [rhel7] Revert: Remove 2 second startup wait.
Bugzilla: 857123
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Vlad Yasevich <vyasevic@redhat.com>
RH-Acked-by: Richard Jones <rjones@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>

Bugzilla: 857123
Upstream: N/A (downstream customization)

This reverts RHEL7 bd2e0975ef2213375bf97bcf2b39552e8a1a591a

Downstream customized the banner timeout to remove it entirely,
allowing users no opportunity for interaction with the iPXE shell.
Remove this in preparation for removing only the banner displayed
during ROM init and keeping the banner displayed prior to autoboot.

Signed-off-by: Alex Williamson <alex.williamson@redhat.com>
---
 src/config/general.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 src/config/general.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/config/general.h b/src/config/general.h
index c497ed3..9f0bb52 100644
--- a/src/config/general.h
+++ b/src/config/general.h
@@ -31,7 +31,7 @@ FILE_LICENCE ( GPL2_OR_LATER );
  * Timer configuration
  *
  */
-#define BANNER_TIMEOUT	0	/* Tenths of a second for which the shell
+#define BANNER_TIMEOUT	20	/* Tenths of a second for which the shell
 				   banner should appear */
 
 /*
-- 
1.7.1

